parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -346,9 +346,11 @@ configureContainerdRegistryHost() {
   mkdir -p "$(dirname "${CONTAINERD_CONFIG_REGISTRY_HOST_MCR}")"
   touch "${CONTAINERD_CONFIG_REGISTRY_HOST_MCR}"
   chmod 0644 "${CONTAINERD_CONFIG_REGISTRY_HOST_MCR}"
+  IFS='/' read -ra container_registry_server <<< "${BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER%/}"
```

Comment:
will it possible it has two level repo? e.g. `xxx.azurecr.io/aks-managed/oss`

Reply:
no, currently, it's not specified by users, it's a fixed value defined by AKS